### PR TITLE
launcherSubmitR convenience method

### PR DESCRIPTION
**Pull request by gtritchie**
Added a convenience method that lets you run an R Script via launcher with minimal parameters:

id <- launcherSubmitR("/home/gary/R/aaa/keepbusy.R")

- removed `launcherStreamStatus` and `launcherStreamOutput` as they aren't useful from R code
- documentation tweaks, job control verbs are case sensitive and must be lower-case

### CODE REVIEW

File: R/launcher-functions.R [hunk 1]
```diff
@@ -295,52 +295,56 @@ launcherSubmitJob <- function(name,
...
+  # don't proactively check for existence of the script; possible the supplied
+  # path only resolves correctly in the job cluster
+  scriptPath <- path.expand(script)
+  scriptFile <- basename(scriptPath)
+  scriptArg <- paste("-f", scriptPath)
+  jobTag <- paste("rstudio-r-script-job", scriptFile, sep = ":")
+  callLauncherFun("launcher.submitJob",
+                  args = c("--slave", "--no-save", "--no-restore", scriptArg),
+                  cluster = cluster,
+                  command = "R",
```
Comment (kevinushey) [inline R/launcher-functions.R]: Maybe make this a parameter so users could select a particular version of R to run with (e.g. if they had already configured their cluster machines to provide R at `/opt/local/R/3.5.2/bin/R` or something)?

File: R/launcher-functions.R [hunk 2]
```diff
@@ -295,52 +295,56 @@ launcherSubmitJob <- function(name,
...
 
+#' Execute an R Script as a Launcher Job
+#'
+#' Convenience function for running an R script as a launcher job using whichever
+#' R is found on the path in the launcher cluster.
+#'
+#' See [launcherSubmitJob()] for running jobs with full control over command,
+#' environment, and so forth.
+#'
+#' @param script Fully qualified path of R script. Must be a path that is
```
Comment (kevinushey) [inline R/launcher-functions.R]: The documentation feels a bit confusing to me. This should be the path to an R script on the user's local machine, is that right? Why does that same path need to exist on the cluster? (E.g. couldn't we just form a 'safe' version of the path for running with the launcher if necessary)?

Odds are I'm just misunderstanding some of the semantics here.

File: R/launcher-functions.R [hunk 3]
```diff
@@ -295,52 +295,56 @@ launcherSubmitJob <- function(name,
...
+#'   available in the job container (if using containerized job cluster such as Kubernetes).
+#' @param cluster The name of the cluster this job should be submitted to.
+#' @param container The container to be used for launched jobs.
```
Comment (kevinushey) [inline R/launcher-functions.R]: Can we document what `NULL` means here?

File: R/launcher-functions.R (hunk 2 above)
Comment (kevinushey) [reply R/launcher-functions.R]: Does this function handle automagically uploading the R script to be run to the cluster?

Comment (gtritchie) [reply R/launcher-functions.R]: This function runs an R script in the same way that the Jobs pane does. It does not upload the script, it requires the script’s location being mounted in the cluster. The pane also uses whichever R is on the path in the cluster/container. Supporting specification of other R versions is not something we are doing for 1.2, but probably for 1.2.x.

Comment (gtritchie) [reply R/launcher-functions.R]: This function runs an R script in the same way that the Jobs pane does. It does not upload the script, it requires the script’s location being mounted in the cluster. The pane also uses whichever R is on the path in the cluster/container. Supporting specification of other R versions is not something we are doing for 1.2, but probably for 1.2.x.

**Pull request merged**